sources/packages/backend/libs/queue/src/constants/queue.constant.ts

```diff
@@ -0,0 +1,14 @@
+import { QueueModel } from "../model/queue.model";
+
+/**
+ * Constant which holds the collection of all the queues.
+ * Any queue which is a part of application must be added here.
+ * Queue modules and bull dashboard use this value to dynamically register the queues.
+ */
+export const Queues = {
```

Comment:
Having this list here makes me think that the queues would be always static in their configurations.
I would prefer having them initialized inside the module or some service. For instance, if we need to configure the queues to run with some corn expressions and maybe get these configurations from the DB, it would be better to have the services injected. In general, it seems something that maybe will be not so static.
From this, the only thing that seems that need to be shared is the `start-application-assessment`, right?
The other applications do not need access to the Queue itself, I guess.